### PR TITLE
Merging js parse

### PR DESCRIPTION
**Pull request by jsarabia**

### CODE REVIEW

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/AppDatabase.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ class AppDatabase(
         dsl = DSL.using(sqLiteDataSource.connection, SQLDialect.SQLITE)
 
         // Check if the database file exists
-        val schemaFile = File(listOf("src", "main", "Resources", "CreateAppDb.sql").joinToString(File.separator))
+        val schemaFile = File(listOf("src", "main", "resources", "CreateAppDb.sql").joinToString(File.separator))
```
Comment (mbr4477) [inline src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/AppDatabase.kt]: I know this isn't relevant to your changes on this branch, but I realized yesterday that this should be changed to something more like: `File(ClassLoader.getSystemResource("CreateAppDb.sql").toURI())`

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/CollectionEntity.kt [hunk 2]
```diff
@@ -8,5 +8,5 @@ data class CollectionEntity(
         var title: String,
         var slug: String,
         var sort: Int,
-        var metadataFk: Int
+        var metadataFk: Int?
```
Comment (mbr4477) [inline src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/CollectionEntity.kt:11]: Are we keeping this as nullable for now?

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/AppDatabase.kt (hunk 1 above)
Comment (jsarabia) [reply src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/AppDatabase.kt]: yeah, for sure

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/CollectionEntity.kt (hunk 2 above)
Comment (jsarabia) [reply src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/CollectionEntity.kt:11]: yeahhhhhhhh unfortunately

**Review by mbr4477 — APPROVED** [review]

**Pull request merged**